File: assets/src/Logger.ts
const Logger = {
    Error: (...message: any) => {
        console.error('%c Millennium ', 'background: red; color: white', ...message);    
    },
    Log: (...message: any) => {
        console.log('%c Millennium ', 'background: purple; color: white', ...message);    
    },
    Warn: (...message: any) => {
        console.warn('%c Millennium ', 'background: orange; color: white', ...message);    
    }
}

export { Logger }

File: src/lua_host/api/types/logger.lua
---@meta

---Logger module for Millennium plugin development
---Provides logging functionality with different severity levels
---@class logger
local logger = {}

---Log an informational message
---@param message string The message to log
function logger:info(message) end

---Log a warning message  
---@param message string The warning message to log
function logger:warn(message) end

---Log an error message
---@param message string The error message to log
function logger:error(message) end

return logger